uint8 command_id
float32 value
uint8 relais
---
bool success
